src/lib/graphqlErrorHandler.ts

```diff
@@ -3,6 +3,7 @@ import { assign } from "lodash"
 import { error as log } from "lib/loggers"
 import { GraphQLTimeoutError } from "lib/graphqlTimeoutMiddleware"
 import { Request } from "../../node_modules/@types/express"
```

Comment:
Is the `../` into `node_modules` from auto-import? I've noticed that VSCode sometimes does this.

Reply:
This is definitely a bug in typescript, was fixed in 3.0 - https://github.com/Microsoft/TypeScript/issues/23230

( which I updated us to on master)